File: taskmanager/app/entities/Comment.go
package entities

type Comment struct {
	Id       int ` json:"id" xml:"id" `
	AuthorId int ` json:"authorid" xml:"authorid" `
	TaskId   int ` json:"taskid" xml:"taskid" `
	Text     string ` json:"text" xml:"text" `
}

File: taskmanager/public/js/src/entities/Comment.js
'use strict'

class Comment {

    #id;
    #authorid;
    #taskid;
    #text;

    constructor(id, authorid, taskid, text) {
        this.#id = id;
        this.#authorid = authorid;
        this.#taskid = taskid;
        this.#text = text;
    }

    getId() {
        return this.#id;
    }

    getAuthorId() {
        return this.#authorid;
    }


    getTaskId() {
        return this.#taskid;
    }

    getText() {
        return this.#text;
    }

    setText(text) {
        this.#text = text;
    }
}